scurl/cgurl.pyx

```diff
@@ -85,7 +76,7 @@ cdef bytes unicode_handling(str):
         bytes_str = <bytes>str
     return bytes_str
```

Comment:
I'm curious how this works: we have ``cdef bytes bytes_str`` above and return it from the function, but return type of the function is ``char *`` - is this correct?

Reply:
Never mind, looks like Cython indeed does this conversion automatically.